connecting to Carbon at 127.0.0.1:2003 (localhost)
zookeeper error on zoo_create, path /rtb-test/serviceClass/monitor/monitor: node exists

--------------------------[Exception thrown]---------------------------
time:   2014-06-24T17:16:20
type:   ML::Exception
pid:    7787; tid: 7787
what:   Zookeeper error on zoo_create, path /rtb-test/serviceClass/monitor/monitor: node exists
stack:
00: 0x0x7fc9a93d1db7 at __cxa_throw + 0x27 in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libexception_hook.so + 0xdb7/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libexception_hook.so(__cxa_throw+0x27) [0x7fc9a93d1db7]
01: 0x0x7fc9a7c9d553 at Datacratic::ZookeeperConnection::checkRes(int, int&, char const*, char const*) + 0xe3 in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so + 0x118553/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic19ZookeeperConnection8checkResEiRiPKcS3_+0xe3) [0x7fc9a7c9d553]
02: 0x0x7fc9a7c9d151 at Datacratic::ZookeeperConnection::createNode(std::string const&, std::string const&, bool, bool, bool, bool) + 0x441 in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so + 0x118151/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic19ZookeeperConnection10createNodeERKSsS2_bbbb+0x441) [0x7fc9a7c9d151]
03: 0x0x7fc9a7c54cd5 at Datacratic::ZookeeperConfigurationService::setUnique(std::string const&, Json::Value const&) + 0x285 in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so + 0xcfcd5/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic29ZookeeperConfigurationService9setUniqueERKSsRKN4Json5ValueE+0x285) [0x7fc9a7c54cd5]
04: 0x0x7fc9a7c3fd84 at Datacratic::ServiceBase::registerServiceProvider(std::string const&, std::vector<std::string, std::allocator<std::string> > const&) + 0x1c4 in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so + 0xbad84/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libservices.so(_ZN10Datacratic11ServiceBase23registerServiceProviderERKSsRKSt6vectorISsSaISsEE+0x1c4) [0x7fc9a7c3fd84]
05: 0x0x7fc9a8d50bbf at RTBKIT::MonitorEndpoint::init(std::vector<std::string, std::allocator<std::string> > const&) + 0x14f in /home/krzysztof/rtbkit/build/x86_64/bin/../bin/libmonitor_service.so + 0x9bbf/home/krzysztof/rtbkit/build/x86_64/bin/../bin/libmonitor_service.so(_ZN6RTBKIT15MonitorEndpoint4initERKSt6vectorISsSaISsEE+0x14f) [0x7fc9a8d50bbf]
06: 0x0x40947f at main + 0x4af in build/x86_64/bin/monitor_service_runner + 0x947fbuild/x86_64/bin/monitor_service_runner(main+0x4af) [0x40947f]
07: 0x0x7fc9a7f2476d at __libc_start_main + 0xed in /lib/x86_64-linux-gnu/libc.so.6 + 0x2176d/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xed) [0x7fc9a7f2476d]
08: 0x0x4099a5 in build/x86_64/bin/monitor_service_runner + 0x99a5build/x86_64/bin/monitor_service_runner() [0x4099a5]

terminate called after throwing an instance of 'ML::Exception'
  what():  Zookeeper error on zoo_create, path /rtb-test/serviceClass/monitor/monitor: node exists
